charts/generic-govuk-app/values.yaml

```diff
@@ -7,15 +7,6 @@ service:
 
 ingress:
   enabled: false
-  annotations:
```

Comment:
These defaults weren't being used anywhere and they would have caused a problem with the addition to ingress.yaml on lines 22-24, hence getting rid of them now. It's also potentially confusing to have AWS-specific stuff being set in the chart defaults, especially when it's actually unused.

Reply:
if i am right in saying the Healthcheck path is used for assigning path in the target group of an App
example:
https://eu-west-1.console.aws.amazon.com/ec2/home?region=eu-west-1#TargetGroup:targetGroupArn=arn:aws:elasticloadbalancing:eu-west-1:210287912431:targetgroup/k8s-apps-servicem-3b7164aa5a/c03dc7568155497f

Reply:
Oh that's a great point — it looks (to my surprise) that those annotations were actually being merged with the per-app ones from `../app-config/values-*.yaml` and we're currently relying on that.

I've put the annotations back in here. (Tried setting them at the top level in `../app-config/values-integration.yaml` but that doesn't work without extra template faff, so I've just stuck a TODO in for now.)

Fixed.

Reply:
It was only really the `rules` that need to go for the purpose of this PR.